frontend/src/components/common/podcast-filter/podcast-filter.tsx

```diff
@@ -2,7 +2,7 @@ import { Path, FieldValues } from 'react-hook-form';
 import { Genre, GenresFilter, PodcastLoadFilter } from 'common/types/types';
```

Comment:
rename to `frontend/src/components/common/podcast-filter-popup/podcast-filter-popup.tsx`